tools/interactive_marker_teleop/src/interactive_marker_teleop.cpp

```diff
@@ -0,0 +1,284 @@
+/* Copyright (c) 2017, United States Government, as represented by the
+ * Administrator of the National Aeronautics and Space Administration.
+ *
+ * All rights reserved.
+ *
+ * The Astrobee platform is licensed under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with the
+ * License. You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+#include <interactive_marker_teleop.h>
+
+#include <string>
+
+InteractiveMarkerTeleop::InteractiveMarkerTeleop(ros::NodeHandle& nh) : nh(nh) {
+  // Start tf listener
+  tfListener = std::shared_ptr<tf2_ros::TransformListener>(new tf2_ros::TransformListener(tfBuffer));
+
+  // Make the marker
+  server.reset(new interactive_markers::InteractiveMarkerServer("interactive_marker_teleop", "", false));
+  ros::Duration(0.1).sleep();
+  menu_handler.insert("Snap to Astrobee",
+                      std::bind(&InteractiveMarkerTeleop::processFeedback, this, std::placeholders::_1));
```

Comment:
Is this best practice for passing member functions to callbacks here?

Reply:
Yup looks good to me